prow/apis/prowjobs/v1/types.go

```diff
@@ -305,6 +305,20 @@ func (d *DecorationConfig) Validate() error {
 	return nil
 }
 
+func (d *DecorationConfig) GetTimeout() *Duration {
+	if d.Timeout == nil {
+		return &Duration{Duration: 0}
+	}
+	return d.Timeout
+}
+
+func (d *DecorationConfig) GetGracePeriod() *Duration {
```

Comment:
nit: Return `time.Duration` instead of `*Duration`. The `time` package type is directly usable and returning a pointer doesn't make much sense since this should always return a concrete value. (This applies to `GetTimeout` as well.)

Reply:
nit: The `GetTimeout` and `GetGracePeriod` functions could be replaced with a single `func (d *Duration) Get() time.Duration` function.

Reply:
addressed